nikola/filters.py

```diff
@@ -146,19 +146,20 @@ def runinplace(command, infile):
 def yui_compressor(infile, executable=None):
     """Run YUI Compressor on a file."""
     yuicompressor = executable
-    if not yuicompressor:
-        try:
-            subprocess.call('yui-compressor', stdout=open(os.devnull, 'w'), stderr=open(os.devnull, 'w'))
-            yuicompressor = 'yui-compressor'
-        except Exception:
-            pass
-    if not yuicompressor:
-        try:
-            subprocess.call('yuicompressor', stdout=open(os.devnull, 'w'), stderr=open(os.devnull, 'w'))
-            yuicompressor = 'yuicompressor'
-        except Exception:
-            raise Exception("yui-compressor is not installed.")
-            return False
+    with open(os.devnull, 'w') as devnull:
```

Comment:
Use [`subprocess.DEVNULL`](https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL) since we only support 3.4+ nowadays.

Reply:
Cool, I was not sure I can use it.